### ISSUE TITLE
onSuccess callback never triggered

### ISSUE THREAD

**Issue by jetaix** [burstware/expo-plaid-link#16]
The onSuccess event is **never** triggered after a successful authentification (from Revolut) and I still stuck on the white screen with the black loader. Any idea ?

**To Reproduce**
1. Select a bank (Revolut)
2. Go to the authentification Revolut flow
3. Success screen from Revolut
4. Redirect to Plaid view
5. No success event triggered / infinite loader screen

**Expected behavior**
Get a onSuccess callback

**Screenshots**
![image](https://user-images.githubusercontent.com/4866709/140439108-402b3ee8-1086-4c7f-a7ad-c100d4132f49.png)

**Expo (please complete the following information):**
 - Version 42.0.0

**Smartphone (please complete the following information):**
 - Device: iPhone6 simulator
 - OS: iOS15

**Comment by JBaczuk** [comment]
@jetaix Are you sure it isn't the bank that is failing to respond? Have you tried another bank? Have you tried it in a web flow?

**Comment by jetaix** [comment]
@JBaczuk Yes I'm sure the bank (Revolut) response a success screen, it's work with another bank, I think it's specific to the Revolut response and how the success url/event is manage by the library.

**Comment by jetaix** [comment]
same issue with a other french bank called `CIC`

**Comment by erikgoins** [comment]
We're having the same issue, the component is returning a HANDOFF event, but not triggering Success.

https://www.loom.com/share/1408eb53bf6d462997f56ef8f604a1be

**Comment by arisopop** [comment]
Hello, I am having having the same issue. I can navigate and authorize in Revolut or Barclays, but in the and is not triggering the onSuccess event.

**Comment by JBaczuk** [comment]
> Hello, I am having having the same issue. I can navigate and authorize in Revolut or Barclays, but in the and is not triggering the onSuccess event.

Do you have any idea if there is any network activity, does their server send a reply?

**Comment by arisopop** [comment]
Yes, the last event is OPEN_AUTH. I authenticate my bank , i see the success message from my bank but after that is not triggering the onSuccess event.

**Comment by arisopop** [comment]
> 

Yes, the last event is OPEN_AUTH. I authenticate my bank , i see the success message from my bank but after that is not triggering the onSuccess event.

**Comment by gurudewan** [comment]
Anyone know if this is fixed? Deciding where I can use this library

**Comment by jordzawada** [comment]
Any word here yet?

**Comment by masiamj** [comment]
Experiencing the same thing. Any thoughts here?

**Comment by JBaczuk** [comment]
I'm sorry I don't have time to try to reproduce this issue. If anyone would like to contribute, I'm happy to take PRs. If anyone would like to take over the project, please let me know.

**Comment by dcolak8figures** [comment]
Same issue here, onSuccess is not being called. No way to finish linking accounts.

**Comment by Laphatize** [comment]
Taking a look at this.
